modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeSetResultsSetTotalMethodCheck.java

```diff
@@ -5,79 +5,176 @@
 		throws Exception {
 
-		if (!fileName.endsWith(".java") && !fileName.endsWith(".jsp") &&
-			!fileName.endsWith(".jspf")) {
+		if (fileName.endsWith(".java")) {
+			content = _formatJava(content, fileName);
+		}
+		else if (fileName.endsWith(".jsp") || fileName.endsWith(".jspf")) {
+			content = _formatContent(content, content, fileName);
+		}
 
-			return content;
+		return content;
+	}
+
+	@Override
+	protected String[] getValidExtensions() {
+		return new String[] {"java", "jsp", "jspf"};
+	}
+
+	private String _formatContent(
+		String content, String fileContent, String fileName) {
+
+		String newContent = fileContent;
+
+		Matcher matcher = _setResultsPattern.matcher(content);
+
+		int setResulstIndex = 0;
+
+		String setResultsMethodCall = null;
+
+		if (matcher.find()) {
+			setResulstIndex = matcher.start();
+
+			setResultsMethodCall = JavaSourceUtil.getMethodCall(
+				content, setResulstIndex);
 		}
 
-		content = _removeSetTotal(content, fileName);
-		content = _replaceSetResults(content, fileName);
+		matcher = _setTotalPattern.matcher(content);
 
-		return content;
+		int setTotalIndex = 0;
+
+		String setTotalMethodCall = null;
+
+		if (matcher.find()) {
+			setTotalIndex = matcher.start();
+
+			setTotalMethodCall = JavaSourceUtil.getMethodCall(
+				content, setTotalIndex);
+		}
+
+		if (!hasClassOrVariableName(
+				"SearchContainer", content, newContent, fileName,
+				setResultsMethodCall) &&
+			!hasClassOrVariableName(
+				"SearchContainer", content, newContent, fileName,
+				setTotalMethodCall)) {
+
+			return newContent;
+		}
+
+		return StringUtil.replace(
+			newContent, content,
+			_getNewMethodCall(
+				content, setResulstIndex, setResultsMethodCall, setTotalIndex,
+				setTotalMethodCall));
 	}
 
-	private String _removeSetTotal(String content, String fileName) {
-		String newContent = content;
+	private String _formatJava(String content, String fileName)
+		throws Exception {
 
-		Matcher setTotalMatcher = _setTotalPattern.matcher(content);
+		JavaClass javaClass = JavaClassParser.parseJavaClass(fileName, content);
 
-		while (setTotalMatcher.find()) {
-			if (hasClassOrVariableName(
-					"SearchContainer", content, content, fileName,
-					JavaSourceUtil.getMethodCall(
-						content, setTotalMatcher.start()))) {
+		if (javaClass == null) {
+			return content;
+		}
 
-				newContent = StringUtil.removeSubstring(
-					newContent, setTotalMatcher.group());
+		for (JavaTerm childJavaTerm : javaClass.getChildJavaTerms()) {
+			if (!childJavaTerm.isJavaMethod()) {
+				continue;
 			}
+
+			JavaMethod javaMethod = (JavaMethod)childJavaTerm;
+
+			String javaMethodContent = javaMethod.getContent();
+
+			content = _formatContent(javaMethodContent, content, fileName);
 		}
 
-		return newContent;
+		return content;
 	}
 
-	private String _replaceSetResults(String content, String fileName) {
+	private String _getNewMethodCall(
+		String content, int setResultIndex, String setResultMethodCall,
```

Comment:
@TamyrisTorres, please see https://github.com/natocesarrego/liferay-portal/pull/1410#discussion_r1395553148

Please make sure this problem is adjusted across all code from this pull :wink: 

Thanks

Reply:
Sorry about this. Now is correct!